core/conn/jdbc_type2/native/SqlInterface.cpp

```diff
@@ -4440,3 +4440,45 @@ void CreateModulePlan(long inputParamCount, InputDescInfo *inputDescInfo, char *
 		}
 	}
 }
+
+SQLRETURN COMMIT_ROWSET(long dialogueId, bool& bSQLMessageSet, odbc_SQLSvc_SQLError* SQLError, Int32 currentRowCount)
+{
+    SQLRETURN retcode;
+    long      sqlcode;
+    SQLValueList_def inValueList;
+    inValueList._buffer = NULL;
+    inValueList._length = 0;
+
+    SRVR_STMT_HDL *CmwSrvrStmt = getInternalSrvrStmt(dialogueId, "STMT_COMMIT_1", &sqlcode);
+    /* Should process the error here if CmwSrvrStmt is NULL */
```

Comment:
For now we make sure this statement is prepared during the connection phase, MXOSRVR do it the same way without a NULL validation, I'm not sure has anyone ever met this kind of problem yet. T2 now has no mechanism to process this kind of null statement error. I'm thinking to return the error back to JAVA layer to throw an exception out, it will be included in the new rowset execute() operation code.